lib/package/sns/compileMethodsToSns.js

```diff
@@ -0,0 +1,147 @@
+'use strict'
+
+const _ = require('lodash')
+
+module.exports = {
+  compileMethodsToSns() {
+    this.validated.events.forEach((event) => {
+      if (event.serviceName == 'sns') {
+        const resourceId = this.getResourceId(event.http.path)
+        const resourceName = this.getResourceName(event.http.path)
+
+        const template = {
+          Type: 'AWS::ApiGateway::Method',
+          Properties: {
+            HttpMethod: event.http.method.toUpperCase(),
+            RequestParameters: {},
+            AuthorizationType: event.http.auth.authorizationType,
+            AuthorizationScopes: event.http.auth.authorizationScopes,
+            AuthorizerId: event.http.auth.authorizerId,
+            ApiKeyRequired: Boolean(event.http.private),
+            ResourceId: resourceId,
+            RestApiId: this.provider.getApiGatewayRestApiId()
+          }
+        }
+
+        _.merge(
+          template,
+          this.getSnsMethodIntegration(event.http),
+          this.getMethodResponses(event.http)
+        )
+
+        const methodLogicalId = this.provider.naming.getMethodLogicalId(
+          resourceName,
+          event.http.method
+        )
+
+        this.apiGatewayMethodLogicalIds.push(methodLogicalId)
+
+        _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources, {
+          [methodLogicalId]: template
+        })
+      }
+    })
+  },
+
+  getSnsMethodIntegration(http) {
+    const integration = {
+      IntegrationHttpMethod: 'POST',
+      Type: 'AWS',
+      Credentials: {
+        'Fn::GetAtt': ['ApigatewayToSnsRole', 'Arn']
+      },
+      Uri: {
+        'Fn::Join': [
```

Comment:
how about `Fn::Sub` here instead? i.e. `Fn::Sub`: 'arn:aws:apigateway:${AWS::Region}:sns:path//'`

Reply:
sure I'll update it